### ISSUE TITLE
Is there a way to save losses only (e.g., val_loss, loss, etc)?

### ISSUE THREAD

**Issue by K-M-Ibrahim-Khalilullah** [matterport/Mask_RCNN#1921]
@moorage @waleedka 
 I would like to save only losses to further analysis after training. Is there any way to save them into a file like .csv or txt.,

Thanks

**Comment by akrsrivastava** [comment]
use CSV Logger from keras callback
